plugins/modules/vm_snapshot_info.py

```diff
@@ -77,17 +77,48 @@
     - A list of VM Snapshot records.
   returned: success
   type: list
-  sample:
-    - automated_trigger_timestamp: 0
-      block_count_diff_from_serial_number: 2
-      label: snap-2
-      local_retain_until_timestamp: 0
-      remote_retain_until_timestamp: 0
-      replication: true
-      snapshot_uuid: 28d6ff95-2c31-4a1a-b3d9-47535164d6de
-      timestamp: 1679397326
-      type: USER
-      vm:
+  elements: dict
+  contains:
+    automated_trigger_timestamp:
+      description: Unix timestamp used when determining which automated snapshots to retain
+      type: int
+      sample: 0
+    block_count_diff_from_serial_number:
+      description: Snapshot serial number of the previous snapshot used to calculate VirDomainSnapshot.blockCountDiff # zakaj nimamo block_count_diff ?
```

Comment:
I will leave question, and add potential answer :). I think the meaning of `block_count_diff_from_serial_number` is `the previous snapshot`. Maybe that by itself has some use (I image create snapshot, restore from it, create a new snapshot, and then you end with non-sequential snapshot serial number.).